client/app/AddressCard.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { set_address } from 'STORE/actions.js';
+import { connect } from 'react-redux';
+import Comments from './Comments';
+
+class AddressCard extends React.Component {
+    render() {
+        const{contact} = this.props
+
+        return(
+            <div className="addressCard">
+                <div className="adressInfo">                
+                    <span>{contanct.address.city}</span>
+                    <span>{contanct.address.street}</span></div>
+                    {contanct.comments.length > 0 &&  <div><Comments /></div>}
```

Comment:
create either a comment list component with the add new component included or map over the comments in here and include the add new comment in here as well